prisma/seeds/players.ts

```diff
@@ -0,0 +1,75 @@
+import type { PrismaClient, Team } from '@prisma/client'
+
+const seedPlayers = (prisma: PrismaClient, teams: Team[]) => {
+  const teamId = teams.find(team => team.key === 'australia')!.id
```

Comment:
given that teams are no longer unique by key, only by key-sportId combination, this should find the team for australia AND rugby